spec/HttpMethodsClientSpec.php

```diff
@@ -74,6 +77,22 @@ function it_sends_a_options_request()
 
         $this->options($data['uri'], $data['headers'], $data['body'])->shouldReturn(true);
     }
+
+    function it_should_send_request_with_underlying_client(HttpClient $client, MessageFactory $messageFactory, RequestInterface $request, ResponseInterface $response)
+    {
+        $client->sendRequest($request)->shouldBeCalled()->willReturn($response);
+
+        $this->beConstructedWith($client, $messageFactory);
+        $this->sendRequest($request)->shouldReturn($response);
+    }
+
+    function it_should_send_requests_with_underlying_client(HttpClient $client, MessageFactory $messageFactory, RequestInterface $request1, RequestInterface $request2, BatchResult $batchResult)
```

Comment:
Can you pleas follow the language conventions above? `it_sends_...`

Thank you